server/autolink/lib_jira_test.go

```diff
@@ -118,6 +118,88 @@ var jiraTests = []linkTest{
+			Template: "[DKHPROJ-${jira_id} With Focused Comment($comment_id)](http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-${jira_id}?focusedCommentId=$comment_id)",
+		},
+		"http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-5?focusedCommentId=10200",
+		"[DKHPROJ-5 With Focused Comment(10200)](http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-5?focusedCommentId=10200)",
+	},
+	{
+		"Jump To Comment With Jira Ecc Long Link",
+		autolink.Autolink{
+			Pattern:  "(http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/)(DKHPROJ)(-)(?P<jira_id>\\d+)[?](focusedCommentId)(=)(?P<comment_id>\\d+)",
+			Template: "[DKHPROJ-${jira_id} With Focused Comment($comment_id)](http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-${jira_id}?focusedCommentId=$comment_id)",
+		},
+		"http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-5?focusedCommentId=10200&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-10200",
+		"http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-5?focusedCommentId=10200&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-10200",
+	},
+	{
+		"Jump To Comment With Jira Ecc Long Short",
+		autolink.Autolink{
+			Pattern:  "(http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/)(DKHPROJ)(-)(?P<jira_id>\\d+)[?](focusedCommentId)(=)(?P<comment_id>\\d+)",
+			Template: "[DKHPROJ-${jira_id} With Focused Comment($comment_id)](http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-${jira_id}?focusedCommentId=$comment_id)",
+		},
+		"http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-5",
+		"http://ec2-54-157-116-101.compute-1.amazonaws.com/browse/DKHPROJ-5",
+	},
+	{
+		"Jump To Comment With Jira Cloud Failed",
+		autolink.Autolink{
+			Pattern:  "(https://mmtest.atlassian.net/browse/)(DP)(-)(?P<jira_id>\\d+)[?](focusedCommentId)(=)(?P<comment_id>\\d+)",
+			Template: "[DP-${jira_id} With Focused Comment($comment_id)](https://mmtest.atlassian.net/browse/DP-${jira_id}",
+		},
+		"https://mmtest.atlassian.net/browse/DP-454",
+		"https://mmtest.atlassian.net/browse/DP-454",
+	},
+	// Trials Linker
+	{
+		"Trial With Jira Cloud",
+		autolink.Autolink{
+			Pattern:  "(?P<URI>.*/)(?P<IssueKey>[A-Za-z]+-[0-9]+)[?](focusedCommentId=*(?P<FocusedCommentId>[^\\s&]+))",
+			Template: "[${IssueKey} FocusedComment(${FocusedCommentId})](${URI}${IssueKey}?focusedCommentId=${FocusedCommentId})",
+		},
+		"https://mmtest.atlassian.net/browse/DP-454?focusedCommentId=11347",
+		"[DP-454 FocusedComment(11347)](https://mmtest.atlassian.net/browse/DP-454?focusedCommentId=11347)",
+	},
+	{
+		"Trial With Jira Server #1",
+		autolink.Autolink{
+			Pattern:  "(?P<URI>.*/)(?P<IssueKey>[A-Za-z]+-[0-9]+)[?](focusedCommentId=*(?P<FocusedCommentId>[^\\s&]+))",
+			Template: "[${IssueKey} FocusedComment(${FocusedCommentId})](${URI}${IssueKey}?focusedCommentId=${FocusedCommentId})",
```

Comment:
I think just putting `(comment)` in the link text is enough to show that it's a comment

```suggestion
			Template: "[${IssueKey} (comment)](${URI}${IssueKey}?focusedCommentId=${FocusedCommentId})",
```